ray-operator/controllers/ray/common/association.go

```diff
@@ -93,6 +103,16 @@ func RayClusterAllPodsAssociationOptions(instance *rayv1.RayCluster) Association
 	}
 }
 
+func RayClusterServicesAssociationOptions(instance *rayv1.RayCluster) AssociationOptions {
```

Comment:
What's the difference between this function and `HeadServiceAssociationOptions`?